openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/TestCaseForm.tsx

```diff
@@ -113,7 +133,7 @@ const TestCaseForm: React.FC<TestCaseFormProps> = ({
 
       setTestCases(data);
     } catch (error) {
-      showErrorToast(error as AxiosError);
+      setTestCases([]);
```

Comment:
Don't we want to show the toast error message here?

Reply:
Yes, we don't need toast here as we are fetching this data to validate duplicate name, even if this API fails, the user should not get error warning